src/screens/docs/index.js

```diff
@@ -1,31 +1,102 @@
 import React from "react";
-import ReactDOM from "react-dom";
-import Ecology from "ecology";
-import GettingStarted from "../../../node_modules/component-playground/docs/getting-started.md";
-import Api from "../../../node_modules/component-playground/docs/api.md";
+import find from "lodash/find";
 
-class Guide extends React.Component {
-  render() {
+import MarkdownIt from "markdown-it";
+import markdownItTocAndAnchor from "markdown-it-toc-and-anchor";
+import Prism from "prismjs";
+/* eslint-disable no-unused-vars */
+// add more language support
+import jsx from "prismjs/components/prism-jsx";
+import sh from "prismjs/components/prism-bash";
+import yaml from "prismjs/components/prism-yaml";
+/* eslint-enable no-unused-vars */
+
+import Page from "../../components/page";
+import basename from "../../basename";
+import { config } from "../../components/config";
+
+
+class Docs extends React.Component {
+  componentDidMount() {
+    Prism.highlightAll();
+  }
+
+  componentDidUpdate() { // is this the right one??
+    Prism.highlightAll();
+  }
+
+  componentWillMount() {
+    this.setMarkdownRenderer(this.props.location.pathname);
+  }
+
+  /* eslint-disable camelcase, max-params */
+  // Create a markdown renderer that builds relative links
+  // based on the currentPath and site's base href
+  setMarkdownRenderer(currentPath) {
+    const md = new MarkdownIt({
+      html: true,
+      linkify: true,
+      typographer: true
+    });
+
+    md.use(markdownItTocAndAnchor, {
+      anchorLinkSymbol: "",
+      anchorClassName: "Anchor",
+      tocCallback(tocMarkdown, tocArray) {
+        // TODO: Pass toc to <Page /> component
+        console.log(tocArray); // eslint-disable-line no-console
```

Comment:
Correct! I didn’t figure it out and didn’t want to continue messing about in this PR.